lib/traiderdaive/traiderdaive/gym_environments/full_hyperdrive_env.py

```diff
@@ -0,0 +1,490 @@
+"""A hyperdrive rl gym environment."""
+
+from __future__ import annotations
+
+import warnings
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any
+
+import gymnasium as gym
+import numpy as np
+import pandas as pd
+from fixedpointmath import FixedPoint
+from gymnasium import spaces
+from scipy.special import expit
+
+from agent0.hyperdrive.interactive import ILocalChain, ILocalHyperdrive
+from agent0.hyperdrive.policies import PolicyZoo
+
+# Global suppression of warnings, TODO fix
+warnings.filterwarnings("ignore")
+
+
+class TradeTypes(Enum):
+    """Enum denoting between long and short indices"""
+
+    LONG = 0
```

Comment:
It's confusing to have this as the list of TradeTypes when we also allow adding/removing LP. Is there a reason you don't include that here?

Reply:
Longs/shorts are handled in the same way, and these enums help differentiate between them. LPs have a different action/observation space and is handled differently

Reply:
ok, mind adding that to the docstring for trade types?